### DISCUSSION TITLE
CUDA 12 or 11.8 for dev?

### DISCUSSION THREAD

**Discussion by BlairSadewitz** [PygmalionAI/aphrodite-engine#150] [Q&A] [answered]
In the instructions for the dev branch, it says both that CUDA 12.0 and CUDA 11.8 are required.  The  yaml specifies 11.8.0, so I'm kinda confused here. ;-)

Is it that you never updated environment.yaml, or that 12.0 isn't actually required?

**Comment by AlpinDale** [comment, marked as answer]
Ah sorry my bad! I meant to say CUDA < 12 in the README. Thanks for pointing that out.

We're sticking with 11.8 and below for maximum compatibility, as 12 would force us to drop support for older GPUs.

**Comment by BlairSadewitz** [comment]
Ah, ok, thanks.  Glad to hear it for that reason, heh.
